trinity/protocol/bcc/servers.py

```diff
@@ -0,0 +1,126 @@
+import functools
+import itertools
+
+from cytoolz import (
+    cons,
+    sliding_window,
+    take,
+)
+from eth_utils import (
+    to_tuple,
+)
+
+from cancel_token import CancelToken
+
+from p2p import protocol
+from p2p.peer import BasePeer
+from p2p.protocol import Command
+
+from eth.exceptions import BlockNotFound
+from eth.beacon.db.chain import BaseBeaconChainDB
+from eth.beacon.types.blocks import BaseBeaconBlock
+
+from trinity.protocol.common.servers import BaseRequestServer
+from trinity.protocol.bcc import commands
+from trinity.protocol.bcc.peer import (
+    BCCPeer,
+    BCCPeerPool,
+)
+
+from typing import (
```

Comment:
nitpick but this should move under itertools as it is part of the standardlib.

Ordering should be:

- standard libs
- installed libs
- local

See also: https://github.com/ethereum/snake-charmers-tactical-manual/blob/master/style-guide.md#overall-ordering